### ISSUE TITLE
Clear Linux sync to cloud.

### ISSUE THREAD

**Issue by nazar2sfive** [clearlinux/distribution#1057]
Requesting for the package "ClearSync" 

Hope with every new fresh install - all my software, gnome extension, personal data, passwords, wallpaper, theme gets autosync to cloud and desktop, just like in ChromeOS.

**Comment by ahkok** [comment]
Let me try and understand this correctly: You're asking us to:

- create a new cloud service (and host it)
- create an end user application

And these would work together so that users can log in and save/retrieve specific user settings from the service?

**Comment by nazar2sfive** [comment]
@ahkok Yes. So that my user settings(software, gnome extensions, theme, wallpaper, personal desktop customization) will be in sync to multiple clear linux desktop.

**Comment by lebensterben** [comment]
>@ahkok Yes. So that my user settings(software, gnome extensions, theme, wallpaper, personal desktop customization) will be in sync to multiple clear linux desktop.

@nazar2sfive Just copy appropriate contents from your home directory to another system. Usually per-user configurations are hidden directories like `~/.gnupg`, or in `~/.config` and `~/.local/`.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
